### PR TITLE
Upgraded Grace reference to pre-release.

### PR DESCRIPTION
**Pull request by Lex45x**
Upgraded GraceServiceProvider and GraceServiceScope to be IAsyncDisposable

### CODE REVIEW

File: src/Grace.DependencyInjection.Extensions/Grace.DependencyInjection.Extensions.csproj.user [hunk 1]
```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (ipjohnson) [inline src/Grace.DependencyInjection.Extensions/Grace.DependencyInjection.Extensions.csproj.user]: I don't think we need this file

**Review by ipjohnson — COMMENTED** [review]
I think if you bump the version to 8.0.0-beta and remove the extra file this is good to go.

File: src/Grace.DependencyInjection.Extensions/Grace.DependencyInjection.Extensions.csproj.user (hunk 1 above)
Comment (Lex45x) [reply src/Grace.DependencyInjection.Extensions/Grace.DependencyInjection.Extensions.csproj.user]: Ah, my bad. Did not noticed this file.

**Comment by Lex45x** [comment]
Done!

**Comment by ipjohnson** [comment]
This is released

**Pull request merged**